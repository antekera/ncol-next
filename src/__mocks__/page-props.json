{
  "pageProps": {
    "preview": false,
    "pageTitle": "Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista",
    "pageType": "/SINGLE",
    "post": {
      "title": "Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista",
      "antetituloNoticia": "Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista",
      "fuenteNoticia": "Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista",
      "slug": "inicio-la-inauguracion-del-mundial-de-qatar-2022",
      "date": "2022-11-20T10:57:15",
      "excerpt": "<p>La espera se terminó: el Mundial Qatar 2022 se puso en marcha con una ceremonia inaugural a pura música y color en el estadio Al Bayt. Más de 60 mil espectadores",
      "featuredImage": {
        "node": {
          "sourceUrl": "https://i0.wp.com/admin.noticiascol.com/wp-content/uploads/2022/11/Screenshot_9-8.jpg?fit=616%2C347&ssl=1",
          "altText": "",
          "caption": null
        }
      },
      "categories": {
        "edges": [
          {
            "node": {
              "name": "_Pos_Destacado",
              "uri": "/category/_pos_destacado/",
              "slug": "_pos_destacado"
            }
          },
          {
            "node": {
              "name": "Copa Mundial Catar 2022",
              "uri": "/category/deportes/copa-mundial-catar-2022/",
              "slug": "copa-mundial-catar-2022"
            }
          },
          {
            "node": {
              "name": "Deportes",
              "uri": "/category/deportes/",
              "slug": "deportes"
            }
          },
          {
            "node": {
              "name": "Futbol",
              "uri": "/category/deportes/futbol/",
              "slug": "futbol"
            }
          }
        ]
      },
      "tags": { "edges": [] },
      "contentType": { "node": { "id": "cG9zdF90eXBlOnBvc3Q=" } },
      "isPreview": false,
      "isRestricted": false,
      "isRevision": false,
      "status": "publish",
      "template": { "templateName": "Default" },
      "uri": "/2022/11/20/inicio-la-inauguracion-del-mundial-de-qatar-2022/",
      "content": "<p>La espera se terminó: el Mundial Qatar 2022 se puso en marcha con una ceremonia inaugural a pura música y color en el estadio Al Bayt. Más de 60 mil espectadores disfrutaron del show de apertura de la cita máxima y observaron de cerca la Copa del Mundo, que fue exhibida por el francés Marcel Desailly. Además, se movieron al ritmo de Jung Kook, líder de la banda surcoreana BTS, y escucharon atentamente las palabras de Morgan Freeman y el jeque Tamim bin Hamad Al Thani.</p>\n<h2 id=\"lbp_a1\">La&#8217;eeb voló por el cielo de Qatar 2022.</h2>\n<p>La mascota oficial del Mundial, fiel representación del mundo árabe, dijo presente en el inicio de la ceremonia.</p>\n<img loading=\"lazy\" class=\"aligncenter\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506990/inauguracion-mundial_416x234.webp?resize=567%2C319&#038;ssl=1\" alt=\"Inauguración Mundial\" width=\"567\" height=\"319\" data-recalc-dims=\"1\" />\n<h2 id=\"lbp_a2\">El desfile de las mascotas históricas del Mundial.</h2>\n<p>En la inauguración de Qatar 2022, pasearon por el campo de juego de Al Bay los viejos personajes representativos de las Copas del Mundo.</p>\n<img loading=\"lazy\" class=\"aligncenter\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506987/desfile-mascotas-mundial_416x234.webp?resize=485%2C273&#038;ssl=1\" alt=\"desfile mascotas Mundial\" width=\"485\" height=\"273\" data-recalc-dims=\"1\" />\n<div class=\"lbp-item\">\n<h2 id=\"lbp_a3\">Jung Kook, de BTS, presente en la inauguración del Mundial Qatar 2022.</h2>\n<p>El vocalista de la popular banda surcoreana le puso música y color al arranque de la Copa del Mundo el Al Bayt.</p>\n<p>&nbsp;</p>\n<figure class=\"rat16x9\"><img loading=\"lazy\" class=\"\" title=\"BTS Mundial Qatar\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1\" srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_862x485.webp 1080w\" alt=\"BTS Mundial Qatar\" width=\"453\" height=\"255\" data-src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1\" data-srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_862x485.webp 1080w\" data-recalc-dims=\"1\" /></figure>\n</div>\n<div class=\"lbp-item\">\n<h2 id=\"lbp_a4\">La Copa del Mundo, en buenas manos.</h2>\n<p>El francés Marcel Desailly, campeón del mundo en Francia 1998, le mostró el trofeo que se disputará en el Mundial de Qatar 2022 a la multitud presente para la apertura y el primer partido, entre el local y Ecuador.</p>\n<p>&nbsp;</p>\n<figure class=\"rat16x9\"><img title=\"Marcel Desailly\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1\" srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506981/marcel-desailly_862x485.webp?v=1 1080w\" alt=\"Marcel Desailly\" data-src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1\" data-srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506981/marcel-desailly_862x485.webp?v=1 1080w\" data-recalc-dims=\"1\" /></figure>\n</div>\n<div class=\"lbp-item\">\n<p>&nbsp;</p>\n</div>\n",
      "customFields": {
        "antetituloNoticia": "lorem ipsum",
        "fuenteNoticia": "tycsports.com",
        "videoNoticia": null
      }
    },
    "content": [
      "<p>La espera se terminó: el Mundial Qatar 2022 se puso en marcha con una ceremonia inaugural a pura música y color en el estadio Al Bayt. Más de 60 mil espectadores disfrutaron del show de apertura de la cita máxima y observaron de cerca la Copa del Mundo, que fue exhibida por el francés Marcel Desailly. Además, se movieron al ritmo de Jung Kook, líder de la banda surcoreana BTS, y escucharon atentamente las palabras de Morgan Freeman y el jeque Tamim bin Hamad Al Thani.</p>",
      "<p\n<h2 id=\"lbp_a1\">La&#8217;eeb voló por el cielo de Qatar 2022.</h2>\n<p>La mascota oficial del Mundial, fiel representación del mundo árabe, dijo presente en el inicio de la ceremonia.</p>\n<img loading=\"lazy\" class=\"aligncenter\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506990/inauguracion-mundial_416x234.webp?resize=567%2C319&#038;ssl=1\" alt=\"Inauguración Mundial\" width=\"567\" height=\"319\" data-recalc-dims=\"1\" />\n<h2 id=\"lbp_a2\">El desfile de las mascotas históricas del Mundial.</h2>\n<p>En la inauguración de Qatar 2022, pasearon por el campo de juego de Al Bay los viejos personajes representativos de las Copas del Mundo.</p>\n<img loading=\"lazy\" class=\"aligncenter\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506987/desfile-mascotas-mundial_416x234.webp?resize=485%2C273&#038;ssl=1\" alt=\"desfile mascotas Mundial\" width=\"485\" height=\"273\" data-recalc-dims=\"1\" />\n<div class=\"lbp-item\">\n<h2 id=\"lbp_a3\">Jung Kook, de BTS, presente en la inauguración del Mundial Qatar 2022.</h2>\n<p>El vocalista de la popular banda surcoreana le puso música y color al arranque de la Copa del Mundo el Al Bayt.</p>\n\n<figure class=\"rat16x9\"><img loading=\"lazy\" class=\"\" title=\"BTS Mundial Qatar\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1\" srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_862x485.webp 1080w\" alt=\"BTS Mundial Qatar\" width=\"453\" height=\"255\" data-src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1\" data-srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_416x234.webp?resize=453%2C255&#038;ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506988/bts-mundial-qatar_862x485.webp 1080w\" data-recalc-dims=\"1\" /></figure>\n</div>\n<div class=\"lbp-item\">\n<h2 id=\"lbp_a4\">La Copa del Mundo, en buenas manos.</h2>\n<p>El francés Marcel Desailly, campeón del mundo en Francia 1998, le mostró el trofeo que se disputará en el Mundial de Qatar 2022 a la multitud presente para la apertura y el primer partido, entre el local y Ecuador.</p>\n\n<figure class=\"rat16x9\"><img title=\"Marcel Desailly\" src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1\" srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506981/marcel-desailly_862x485.webp?v=1 1080w\" alt=\"Marcel Desailly\" data-src=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1\" data-srcset=\"https://i0.wp.com/media.tycsports.com/files/2022/11/20/506981/marcel-desailly_416x234.webp?ssl=1 480w, https://media.tycsports.com/files/2022/11/20/506981/marcel-desailly_862x485.webp?v=1 1080w\" data-recalc-dims=\"1\" /></figure>\n</div>\n<div class=\"lbp-item\">\n\n</div>\n"
    ],
    "posts": {
      "edges": [
        {
          "node": {
            "excerpt": "<p>La espera se terminó: el Mundial Qatar 2022 se puso en marcha con una ceremonia inaugural a pura música y color en el estadio Al Bayt. Más de 60 mil espectadores",
            "title": "Brasil: De varios disparos asesinan a venezolano.(video)",
            "slug": "brasil-de-varios-disparos-asesinan-a-venezolano-video",
            "date": "2022-11-20T09:45:41",
            "featuredImage": {
              "node": {
                "sourceUrl": "https://i0.wp.com/admin.noticiascol.com/wp-content/uploads/2022/11/Screenshot_8-10.jpg?fit=386%2C224&ssl=1",
                "altText": "",
                "caption": null
              }
            },
            "categories": {
              "edges": [
                {
                  "node": {
                    "name": "_Pos_Columna_izq",
                    "uri": "/category/_pos_columna_izq/",
                    "slug": "_pos_columna_izq"
                  }
                },
                {
                  "node": {
                    "name": "Internacionales",
                    "uri": "/category/internacionales/",
                    "slug": "internacionales"
                  }
                },
                {
                  "node": {
                    "name": "Sucesos",
                    "uri": "/category/sucesos/",
                    "slug": "sucesos"
                  }
                }
              ]
            },
            "tags": { "edges": [] },
            "contentType": { "node": { "id": "cG9zdF90eXBlOnBvc3Q=" } },
            "isPreview": false,
            "isRestricted": false,
            "isRevision": false,
            "status": "publish",
            "template": { "templateName": "Default" },
            "uri": "/2022/11/20/brasil-de-varios-disparos-asesinan-a-venezolano-video/"
          }
        },
        {
          "node": {
            "title": "Municipio Maracaibo: Accidente de transito deja como saldo un motorizado muerto",
            "slug": "municipio-maracaibo-accidente-de-transito-deja-como-saldo-un-motorizado-muerto",
            "date": "2022-11-20T09:39:55",
            "featuredImage": {
              "node": {
                "sourceUrl": "https://i0.wp.com/admin.noticiascol.com/wp-content/uploads/2022/07/morgue-hospial-de-maracaibo-2.jpg?fit=700%2C400&ssl=1",
                "altText": "",
                "caption": null
              }
            },
            "categories": {
              "edges": [
                {
                  "node": {
                    "name": "_Pos_Columna_izq",
                    "uri": "/category/_pos_columna_izq/",
                    "slug": "_pos_columna_izq"
                  }
                },
                {
                  "node": {
                    "name": "Maracaibo",
                    "uri": "/category/zulia/maracaibo/",
                    "slug": "maracaibo"
                  }
                },
                {
                  "node": {
                    "name": "Sucesos",
                    "uri": "/category/sucesos/",
                    "slug": "sucesos"
                  }
                }
              ]
            },
            "tags": { "edges": [] },
            "contentType": { "node": { "id": "cG9zdF90eXBlOnBvc3Q=" } },
            "isPreview": false,
            "isRestricted": false,
            "isRevision": false,
            "status": "publish",
            "template": { "templateName": "Default" },
            "uri": "/2022/11/20/municipio-maracaibo-accidente-de-transito-deja-como-saldo-un-motorizado-muerto/"
          }
        }
      ]
    },
    "ads": {
      "menu": {
        "id": "div-gpt-ad-1662938702901-0",
        "style": { "minWidth": "1000px", "minHeight": "250px" }
      },
      "menu_mobile": {
        "id": "div-gpt-ad-1663287871941-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "sidebar": {
        "id": "div-gpt-ad-1662933959123-0",
        "style": { "minWidth": "300px", "minHeight": "600px" }
      },
      "sidebar2": {
        "id": "div-gpt-ad-1662937413122-0",
        "style": { "minWidth": "300px", "minHeight": "250px" }
      },
      "cover": {
        "id": "div-gpt-ad-1663288350704-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "homeFeed": {
        "id": "div-gpt-ad-1662935751586-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "homeFeed2": {
        "id": "div-gpt-ad-1662935053350-0",
        "style": { "minWidth": "200px", "minHeight": "200px" }
      },
      "homeFeed3": {
        "id": "div-gpt-ad-1663438486397-0",
        "style": { "minWidth": "200px", "minHeight": "200px" }
      },
      "categoryFeed": {
        "id": "div-gpt-ad-1663289413827-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "categoryFeed2": {
        "id": "div-gpt-ad-1663289569751-0",
        "style": { "minWidth": "300px", "minHeight": "100px" }
      },
      "squareC1": {
        "id": "div-gpt-ad-1663290647151-0",
        "style": { "minWidth": "300px", "minHeight": "250px" }
      },
      "squareC2": {
        "id": "div-gpt-ad-1663290682160-0",
        "style": { "minWidth": "300px", "minHeight": "250px" }
      },
      "squareC3": {
        "id": "div-gpt-ad-1663290699468-0",
        "style": { "minWidth": "300px", "minHeight": "250px" }
      }
    }
  }
}
