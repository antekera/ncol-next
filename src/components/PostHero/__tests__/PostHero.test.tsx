import React from 'react'

import { render } from '@testing-library/react'

import { PostHero } from '..'
import { pageProps } from '../../../__mocks__/page-props.json'

describe('PostHero', () => {
  test('should match snapshots', () => {
    const { container } = render(
      <PostHero
        categories={pageProps.post.categories}
        title={pageProps.post.title}
        slug={pageProps.post.slug}
        uri={pageProps.post.uri}
        date={pageProps.post.date}
      />
    )
    expect(container.firstChild).toMatchSnapshot()
  })
})
