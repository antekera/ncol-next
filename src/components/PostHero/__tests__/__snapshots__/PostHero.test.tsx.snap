// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostHero should match snapshots 1`] = `
<section>
  <div
    class="relative w-auto px-5 py-2 -mt-10 -ml-6 bg-white border-t-4 sm:ml-0 sm:w-11/12 content border-primary z-2"
  >
    <a
      aria-label="Copa Mundial Catar 2022"
      class="relative inline-block leading-none mr-2 text-xs link-post-category mb-3 uppercase text-primary"
      href="/categoria/copa-mundial-catar-2022"
    >
      Copa Mundial Catar 2022
    </a>
    <h1
      class="mb-2 font-serif text-2xl font-bold leading-tight lg:text-4xl text-slate-900"
    >
      <a
        aria-label="Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista"
        class="hover:text-primary"
        href="/2022/11/20/inicio-la-inauguracion-del-mundial-de-qatar-2022"
      >
        Mundial de Qatar 2022: Se dio inicio a la fiesta mundialista
      </a>
    </h1>
    <hr
      class="relative w-48 mt-4 mb-3 md:w-80 text-slate-200"
    />
    <div
      class="mb-4 text-xs text-lg md:mb-0"
    >
      <time>
        <span
          class="capitalize"
        >
          noviembre 20, 2022
        </span>
        <span>
           â€¢ 10:57 a.m.
        </span>
      </time>
    </div>
  </div>
</section>
`;
